src/main/java/nextstep/courses/domain/CoverFileSize.java

```diff
@@ -0,0 +1,17 @@
+package nextstep.courses.domain;
+
+public class CoverFileSize {
+    private static final long MAX_SIZE = 1024 * 1024; // 1MB
+    private long size;
+
```

Comment:
```suggestion
public class CoverFileSize {
    private static final long MAX_SIZE = 1024 * 1024; // 1MB

    private final long size;
```

값 객체네요!
불변을 유지할 수 있도록 final 필드로 선언하면 좋을 것 같고,
동등성을 보장하기 위해 equals, hashCode 메소드를 오버라이딩하면 좋을 것 같아요 😄

Reply:
파일 사이즈 바이트 단위라도 int 만큼의 범위는 충분하다고 생각하는데 어떻게 생각하시나요?

Reply:
이미지 커버기도 하고 사이즈도 크지 않을 것으로 보아 int가 더 적절해보이네요.
반영하겠습니다!